core/src/androidMain/kotlin/com/ustadmobile/core/impl/UstadMobileSystemImpl.kt

```diff
@@ -485,6 +485,20 @@ actual open class UstadMobileSystemImpl : UstadMobileSystemCommon() {
         (context as Context).startActivity(intent)
     }
 
+    actual fun handleClickLink(url: String, context: Any){
```

Comment:
This code should be in common. The logic will be the same.

Needs jvm tests.

Needs to handle edge case: when link is an ustad link, but a different endpoint